README.md

```diff
@@ -0,0 +1,49 @@
+# Mount Recorder Server
+
+Created by Web Static Initiative members:
+
+- [Sofia Lee](https://www.linkedin.com/in/sofia-lee-58b75114b/)
+- [Evan Monroy](https://www.linkedin.com/in/evan-monroy-917b4a162/)
+
+## Table of contents
+
+- [General info](#general-info)
+- [Technologies](#technologies)
+- [Setup](#setup)
+
+## General info
+
+- This project is the back-end for the mount recorder application.
+- It stores users, mounts, and which mounts users own.
+- It returns data when recieving SQL statements.
```

Comment:
This is a pretty wrong explanation of what the API does.
- It does not `store users, mounts, and which mounts users own`. That is the database's job.
- It does not receive `SQL statement`. It receives API requests.


Just put this:
```
- The project is the back-end API used in [Mount Recorder Client](https://github.com/sofia819/mount_recorder_client).
- On each API request from the client, the corresponding operations are executed and data is returned when necessary.
```

Reply:
I'm honestly amazed I forgot something so foundational and critical. The corrections have been made.